types/grid.ts

```diff
@@ -31,3 +41,53 @@ export enum WidgetType {
   // Eg: Youtube, Google Maps, Spotify, etc
   IFRAME = 'IFRAME',
 }
+
+export const GRID_WIDGET_TYPES_SIZES: GridWidgetTypeSizes = {
+  [GridWidgetType.TITLE_LINK]: {
+    [GridWidgetSize.SMALL]: { w: 1, h: 5 },
+    [GridWidgetSize.MEDIUM]: { w: 2, h: 5 },
+    [GridWidgetSize.LARGE]: { w: 3, h: 5 },
+    [GridWidgetSize.FULL]: { w: 4, h: 5 }
+  },
```

Comment:
I wonder if this spectrum of sizes is not limiting us to much, I would consider more scaleable solution based on the WxH ratio:

```
[GridWidgetType.TITLE_LINK]: ['1x5', '2x5', '3x5'...]
```

This way we can easily add other ratios if needed for instance `1x1`, `1x3`...

Reply:
agree, this has been updated